docs/Plugins.rst

```diff
@@ -37,22 +37,70 @@ For more information, see `the full Stonesense README <stonesense>`.
 
 blueprint
 =========
-Exports a portion of your fortress into QuickFort style blueprint files.
+The ``blueprint`` command exports the structure of a portion of your fortress in
+a blueprint file that you (or anyone else) can later play back with `quickfort`.
 
-Usage::
+Blueprints are ``.csv`` or ``.xlsx`` files created in the ``blueprints``
+subdirectory of your DF folder. The map area to turn into a blueprint is either
+selected interactively with the ``blueprint gui`` command or, if the GUI is not
+used, starts at the active cursor location and extends right and down for the
+requested width and height.
+
+Usage:
+
+    blueprint <width> <height> [<depth>] [<name> [<phases>]] [<options>]
+    blueprint gui [<name> [<phases>]] [<options>]
+
+Examples:
+
+blueprint gui
+    Runs `gui/blueprint`, the interactive blueprint frontend, where all
+    configuration for a ``blueprint`` command can be set visually and
+    interactively.
 
-    blueprint <x> <y> <z> <name> [dig] [build] [place] [query]
+blueprint gui bedrooms dig -\-cursor 108,100,150
+    Starts ``gui/blueprint`` with some configuration options preset to custom
+    values.
 
-Options (If only region and name are given, export all):
+blueprint 30 40 bedrooms
```

Comment:
Perhaps an example of phases + options being used _without_ the `gui` argument would also be useful.

Reply:
actually, let me repurpose the gui example for this. I haven't implemented option parsing for `gui/quickfort` yet, and if we're going to cut a release after this merge, I don't want the docs to be misleading.